packages/voice/src/app/agent/page.tsx

```diff
@@ -221,7 +221,7 @@ const AgentPageComponent: React.FC = () => {
   const ttsVoice = searchParams.get('ttsVoice') || agentVoice;
   const model = getAgent(agentId) === undefined ? 'fixie' : searchParams.get('llm') || DEFAULT_LLM;
   const docs = searchParams.get('docs') !== null;
-  const webrtc = searchParams.get('webrtc') !== null;
+  const webrtcUrl = searchParams.get('webrtc') ?? undefined;
```

Comment:
Could you clarify this for me? My expectation is that with this change we'll have:

"/" => client-hosted
"/?webrtc" => server-hosted with default urls
"/?webrtc=wss://some-gcr-url" => server-hosted with an arbitrary deployment

Is that right?

Reply:
Yeah, I just wanted to be able to test a local FE with a staging deployment.